rclcpp/src/rclcpp/intra_process_manager.cpp

```diff
@@ -12,25 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "rclcpp/experimental/intra_process_manager.hpp"
-
 #include <atomic>
 #include <memory>
 #include <mutex>
 
+#include "rclcpp/experimental/intra_process_manager.hpp"
```

Comment:
nitpick: the header file of a cpp file is ok to be at the top

Reply:
Restored in [2dec8e3](https://github.com/ros2/rclcpp/pull/1847/commits/2dec8e39a7c772dd937a084c803d8fc7c204c18e)